types/screener-runner/index.d.ts

```diff
@@ -0,0 +1,103 @@
+//
+// Typings for `screener-runner` plus some custom things
+//
+
+declare module 'screener-runner' {
```

Comment:
How will this impact Screener tests in this repo currently (if any)?

Reply:
It doesn't change anything, I just restructured the existing file and how it's included.